test/intercept/XMLHttpRequest/XMLHttpRequest.test.ts

```diff
@@ -281,3 +281,18 @@ test('intercepts an HTTPS HEAD request', async () => {
     'x-custom-header': 'yes',
   })
 })
+
+test('ready state names', () => {
```

Comment:
Do you mind moving this test suite into a new folder called `test/compliance`? This one doesn't directly concern request interception. Thanks.